docs/manual/about/under-the-hood/guarantees.md

```diff
@@ -20,15 +20,22 @@ support specific guarantees.
 
 ### At-Least-Once
 
-The `at-least-once` delivery guarantee ensures that an [event][docs.data-model]
-received by a Vector component will be delivered at least once. While rare, it
-is possible for an event to be delivered more than once. See the
-[Does Vector support exactly once delivery](#does-vector-support-exactly-once-delivery)
-FAQ below).
+The `at_least_once` delivery guarantee ensures that an
+[event][docs.data-model] received by a Vector component will be
+delivered at least once. For a source, this indicates that it will wait
+for _all_ connected sinks to mark the event as delivered before
+acknowledging receipt of the event. For a sink, this indicates that it
+will attempt to retry the delivery until the events are either accepted
+or rejected and then signal the source with the results of that
+delivery.
```

Comment:
This isn't the case when buffers are used, right? It will acknowledge once it has been written to the buffer?

Reply:
Correct.